backend/routes/deepface_routes.py

```
@@ -0,0 +1,1316 @@
+
+
+def _set_alert_cooldown(anomaly_type: str, stream_id: str, entity_key: str) -> None:
+    r = _get_alert_redis()
+    if not r:
+        logger.warning("Cooldown set skipped — Redis unavailable")
+        return
+    key = _cooldown_key(anomaly_type, stream_id, entity_key)
+    try:
+        result = r.set(key, "1", ex=settings.ALERT_COOLDOWN_SECONDS, nx=True)
+        logger.debug("Cooldown set: key=%s ttl=%s result=%s", key, settings.ALERT_COOLDOWN_SECONDS, result)
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)